tools/integration_tests/util/setup/setup.go

```diff
@@ -382,6 +382,18 @@ func SetupTestDirectory(testDirName string) string {
 	return testDirPath
 }
 
+// SetupTestDirectoryRecursive recursively creates a testDirectory in the mounted directory and cleans up
+// any content present in it.
+func SetupTestDirectoryRecursive(testDirName string) string {
```

Comment:
Should we pass another boolean variable (recursive) in the existing method `SetupTestDirectory` ?

Although, this looks more readable to me on the cost of duplicity. So, we can keep as it is.

Reply:
I thought about that but, as you mentioned, I did not want to clutter the existing function hence I chose to write a different one. Lets keep it as it is for clarity purpose.